xsqlvar.go

```diff
@@ -310,122 +310,118 @@ func (x *xSQLVAR) parseString(raw_value []byte, charset string) interface{} {
 	if x.sqlsubtype == 1 { // OCTETS
 		return raw_value
 	}
-	if x.sqlsubtype == 0 {
```

Comment:
The value of x.sqlsubtype was found to be 44 instead of 0, so I have modified that part accordingly.
Please review this change.